src/components/about/sections/ManagementBoardSection/ManagementBoardDescription.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@mui/material'
+import { Description } from 'components/about/AboutPage.styled'
+import { useTranslation } from 'next-i18next'
+import { useEffect, useState } from 'react'
+
+export function ManagementBoardDescription({ description }) {
+  const { t } = useTranslation('about')
+
+  const [show, setShow] = useState(false)
```

Comment:
This is a fairly simple component, but I would suggest to put a bit more descriptive name for this toggle, in case we extend it and add more complicated logic in the future, something like:
- `isFullDescriptionVisible`
- `isDescriptionCollapsed`
- etc.